src/Kafka/CreateKafkaTopic/components/CreateTopicHead.tsx

```diff
@@ -0,0 +1,71 @@
+import type { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  PageSection,
+  PageSectionVariants,
+  Title,
+  Switch,
+  Breadcrumb,
+  BreadcrumbItem,
+} from "@patternfly/react-core";
+import { Link } from "react-router-dom";
+
+export type CreateTopicProps = {
+  isSwitchClicked: boolean;
+  kafkaName?: string;
+  kafkaPageLink?: string;
+  kafkaInstanceLink?: string;
```

Comment:
```suggestion
  kafkaName: string;
  kafkaPageLink: string;
  kafkaInstanceLink: string;
```

We don't handle the undefined, so we shouldn't allow for that.